modules/402-ingress-nginx/monitoring/grafana-dashboards/ingress-nginx/namespace/namespace_detail.json

```diff
@@ -10167,14 +10167,36 @@
         "useTags": false
       },
       {
-        "description": null,
-        "error": null,
-        "hide": 2,
+        "current": {
+          "selected": true,
+          "text": "without default backend",
+          "value": "[^_]*"
+        },
+        "hide": 0,
+        "includeAll": false,
         "label": "VHost",
+        "multi": false,
         "name": "vhost",
-        "query": ".*",
+        "options": [
```

Comment:
Didn't get the idea here.

Reply:
No it's not. Below is the code `"query": "[^_]*,.*,_",`